risc0/zkvm/r0vm/r0vm.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 Risc0, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <stdio.h>
+
+#include "risc0/zkvm/sdk/cpp/host/receipt.h"
+
+using namespace risc0;
+
+constexpr size_t kMaxStdin = 64 * 1024;
+
+int main(int argc, char* argv[]) {
+  // Check args
+  if (argc != 2) {
+    fprintf(stderr, "Usage: r0vm <elf_file>\n");
```

Comment:
Use C++ std::cerr instead of fprintf

Reply:
Due to a lot of reading/writing that is more annoying to handle in iostream form, I'm using old school C APIs elsewhere, so I'm just trying to be consistent.